packages/ui/src/ui/pages/chyt/ChytCliqueActions/ChytCliqueActions.tsx

```diff
@@ -49,13 +49,15 @@ export function ChytCliqueActions({
     allButtonsClassName,
     onAction,
     onSqlClick,
+    color,
 }: {
     showAllButtons?: boolean;
     allButtonsClassName?: string;
     alias: string;
     pool?: string;
     onSqlClick: (alias: string) => void;
     onAction?: (action: 'remove' | 'start' | 'stop') => void;
+    color?: 'secondary';
```

Comment:
I suggest renaming the parameter. Based on the type, it can now only be "undefined" or "secondary". The current name "color" implies a color palette